spec/acceptance/deb_admin_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper_acceptance'
+
+describe 'Scenario: pulp-admin-deb' do
+  let(:pp) do
+    <<-EOS
```

Comment:
It might need (like any non-default plugin) https://github.com/theforeman/puppet-pulp/blob/e25020a7bce87fbec849a20cc0df5f7261600eb8/examples/pulp_puppetrepo.pp#L9-L17 because it's likely another test is running before it - doing the db sync. Then pulp can't do the DB migration when services are running.